airbyte-cdk/bulk/toolkits/load-object-storage/src/main/kotlin/io/airbyte/cdk/load/file/object_storage/ObjectStorageFormattingWriter.kt

```diff
@@ -36,9 +36,9 @@ import java.io.Closeable
 import java.io.OutputStream
 import org.apache.avro.Schema
 
-interface ObjectStorageFormattingWriter : Closeable {
-    fun accept(record: DestinationRecord)
-    fun flush()
+abstract class ObjectStorageFormattingWriter : Closeable {
```

Comment:
What is the purpose here? Should we revert this?

Reply:
Hmmm, I thought that I clean this. It is not necessary